packages/@react-spectrum/progress/docs/ProgressCircle.mdx

```diff
@@ -0,0 +1,102 @@
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/progress';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {ProgressCircle} from '@react-spectrum/progress';
+```
+
+# ProgressCircle
+
+<p>{docs.exports.ProgressCircle.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ProgressCircle']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/circle-loader/'}
+  ]} />
+
+## Example
+
+```tsx example
+<ProgressCircle aria-label="Loading…" isIndeterminate />
+```
+
+## Value
+
+By default Progress Circles are controlled with the `value` prop representing current percentage (as an integer).
+In this case, the minimum and maximum values default to 0 and 100, respectively.
+
+```tsx example
+<ProgressCircle aria-label="Loading…" value={25} />
+```
+
+Alternatively, a different scale can be used by setting the `minValue` and `maxValue`.
+
+```tsx example
+<ProgressCircle aria-label="Loading…" minValue={50} maxValue={150} value={100} />
+```
+
+### Accessibility
+
+ProgressCircle requires an `aria-label` or `aria-labelledby` attribute.
+
+```tsx example
+<div>
+  <ProgressCircle aria-labelledby="pc-label" isIndeterminate />
+  <label id="pc-label" style={{display: 'block'}}>Loading…</label>
+</div>
+```
+
+If using the over background variant, ensure the background offers enough contrast for the loader to be legible and meets color contrast guidelines.
+
+### Internationalization
+
+To internationalize a Progress Circle, a localized string should be passed to the `aria-label` prop (or value of associated `aria-labelledby` element).
+
+For RTL (right-to-left) languages, the fill of an both determinate and indeterminate Progress Circles continues to spin clockwise.
```

Comment:
In other examples we say:
"For languages that are read right to left (e.g. Hebrew and Arabic)" ... blah blah

Do you think this is too verbose and rather opt for what you have here? (i.e. "For RTL (right-to-left) languages")

Reply:
Do we need to say this? It's kind of just an implementation detail not something the user really need to worry about.

Reply:
i have removed.